src/app/pages/home/profile/profile.page.html

```diff
@@ -6,15 +6,6 @@
         <img ng-show="isFrontCaptured" [src]="card.frontImg" alt=""/>
     </ion-card>
 
-    <ion-card button style="height:172px;" [routerLink]="['/user', user.id]" *ngFor="let user of (users | async)" lines="none">
```

Comment:
I think this code bit will be useful when we test populate the rolodex for the infinite loop. You with me?